rest_live/decorators.py

```diff
@@ -1,16 +1,20 @@
 from typing import Optional
 
 from django.db.models import Model
+from django.db.models.signals import post_delete, post_save
 from rest_framework import serializers
 
 from rest_live import PermissionLambda, __model_to_listeners
+from rest_live.signals import ondelete_callback, onsave_callback
 
 
 def __register_subscription(
     cls, group_key, check_permission: Optional[PermissionLambda]
 ):
     model: Model = cls.Meta.model
     label = model._meta.label  # noqa
+    post_save.connect(onsave_callback, model, dispatch_uid="rest-live")
```

Comment:
I'm worried that this will register multiple callbacks with multiple decorators. Can we put these inside of the `if label not in __model_to_listeners` check on line 14?

Reply:
The [dispatch uid](https://docs.djangoproject.com/en/3.1/topics/signals/#preventing-duplicate-signals) should guarantee that, I think.